### DISCUSSION TITLE
I can't able to add configuration to run this project after cloing

### DISCUSSION THREAD

**Discussion by NitinSigilipelli** [Ijaiswalshivam/Talk-In#27] [Q&A] [answered]
Is anybody here started running this project may help me to solve this issue at my end 
![error](https://github.com/Ijaiswalshivam/Talk-In/assets/140246876/89d66f66-014d-4218-8ced-a1b364ddbbe4)
Error: Run configuration Unnamed is not supported in the curent project. Cannot obtain the package.

Actually when i clone some other project it automatically detects the configuration and all but here for this project i need to add configuration manually and it shows error 

@Ijaiswalshivam could  you please solve this error how to add configuration for this project

**Comment by Vishwajith-Shettigar** [comment]
Can you show me your directory structure, which is in android studio left panel ?

**Comment by Vishwajith-Shettigar** [comment, marked as answer]
You might be doing same mistake, refer this [discussion](https://github.com/Ijaiswalshivam/Talk-In/discussions/14#discussioncomment-9394461), may help you.

**Comment by NitinSigilipelli** [comment]
thanks @Vishwajith-Shettigar
